themes/tiddlywiki/vanilla/base.tid

```diff
@@ -783,6 +783,10 @@ button svg.tc-image-button, button .tc-image-button img {
 	margin-right: 7px;
 }
 
+.tc-tag-missing {
```

Comment:
I do not think that missing tags should be italicised by default. In many situations, the presence or absence of the tag tiddler is just a consequence of whether the tagged tiddlers have been manually reordered through drag and drop. I do see that some people would want it as an option, but I don't think the core should provide a UI for it as an option.